.kokoro/publish-docs.sh

```diff
@@ -19,15 +17,25 @@ python3.6 -m nox --version
 # build docs
 nox -s docs
 
-cd docs/_build/html
-
 python3 -m pip install gcp-docuploader
 
+# install a json parser
+sudo apt-get update
+sudo apt-get -y install software-properties-common
+sudo add-apt-repository universe
```

Comment:
`add-apt-repository` needs to be before `apt-get update`, or else the new repo won't be used to install anything.

Reply:
The `apt-get update` is to install the command `add-apt-repository`. Otherwise I get:

`sudo: add-apt-repository: command not found`

Once our doc publishing tool can accept this metadata format directly this won't be needed.

Reply:
OK.  I still think you need to re-run `apt-get update` *after* running `add-apt-repository`, or else you won't be able to install anything from the new repository.